### PR TITLE
handling of REAL type / test cases for sql schemas

### PR DESCRIPTION
**Pull request by jgaleotti**

### CODE REVIEW

File: client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.evomaster.clientJava.controllerApi.dto.database.schema.TableDto;
+
+import java.util.List;
+
+public class TableDtoUtils {
+
+
+    public static boolean containsTable(List<TableDto> tables, String tableName) {
+        for (TableDto tableDto :
+                tables) {
```
Comment (arcuri82) [inline client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java]: why the new line for just "table"?

File: client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            if (tableDto.name.equalsIgnoreCase(tableName)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static TableDto getTable(List<TableDto> tables, String tableName) {
+        for (TableDto tableDto :
+                tables) {
```
Comment (arcuri82) [inline client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java]: same comment as above

File: client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java (hunk 1 above)
Comment (jgaleotti) [reply client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java]: the code-autocompletion played a trick on me. I will fix it.

File: client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java (hunk 2 above)
Comment (jgaleotti) [reply client-java/controller/src/test/java/org/evomaster/clientJava/controller/internal/db/TableDtoUtils.java]: same as above

**Comment by jgaleotti** [comment]
I refactored the test cases of the sql schemas to core and put SQL command in a sql file.

**Pull request merged**